com.imageline.flstudio.yml

```diff
@@ -0,0 +1,160 @@
+  - name: platform-setup
+    buildsystem: simple
+    build-commands:
+      - mkdir -p lib/32bit
+      - mkdir -p lib/wine
+
+  - name: wine-win64
+    only-arches:
+      - x86_64
+    config-opts:
+      - --enable-win64
+      - --disable-win16
+      - --disable-tests
+      - --with-x
+      - --without-cups
+      - --without-curses
+      - --without-capi
+      - --without-glu
+      - --without-gphoto
+      - --without-gsm
+      - --without-hal
+      - --without-netapi
+      - --without-opencl
+      - --without-pcap
+      - --without-udev
+      - --without-v4l
+    sources:
+      - type: archive
+        url: https://github.com/wine-mirror/wine/archive/wine-4.13.zip
+        sha256: fcdbb323010182c1650a0f1ad8c181b4f115347bb846ac3808bc8bfc0dad1bb8
+
+  - name: wine-win32
+    config-opts:
+      - --disable-win64
+      - --disable-win16
+      - --disable-tests
+      - --with-x
+      - --without-cups
+      - --without-curses
+      - --without-capi
+      - --without-glu
+      - --without-gphoto
+      - --without-gsm
+      - --without-hal
+      - --without-netapi
+      - --without-opencl
+      - --without-pcap
+      - --without-udev
+      - --without-v4l
+    cleanup:
+      - /bin/function_grep.pl
+      - /include
+      - /share/man
+      - /share/applications
+    sources:
+      - type: archive
+        url: https://github.com/wine-mirror/wine/archive/wine-4.13.zip
+        sha256: fcdbb323010182c1650a0f1ad8c181b4f115347bb846ac3808bc8bfc0dad1bb8
+
+  - name: setup-compat32
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/32bit
+      - ln -s /app/lib/32bit/lib/ld-linux.so.2 /app/lib/ld-linux.so.2
+
+  - name: ensure-gl
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/GL
+
+  - name: flstudio
+    buildsystem: simple
+    build-commands:
+      - install -d /app/bin
+      - install flstudio /app/bin
+      - install flstudio-installer /app/bin
+      - install -Dm644 com.imageline.flstudio.appdata.xml /app/share/appdata/com.imageline.flstudio.appdata.xml
+      - install -Dm644 com.imageline.flstudio.desktop /app/share/applications/com.imageline.flstudio.desktop
+      - install -Dm644 com.imageline.flstudio.256x256.png /app/share/icons/hicolor/256x256/apps/com.imageline.flstudio.png
+      - install -Dm644 com.imageline.flstudio.128x128.png /app/share/icons/hicolor/128x128/apps/com.imageline.flstudio.png
+      - install -Dm644 com.imageline.flstudio.64x64.png /app/share/icons/hicolor/64x64/apps/com.imageline.flstudio.png
+      - install -Dm644 com.imageline.flstudio.48x48.png /app/share/icons/hicolor/48x48/apps/com.imageline.flstudio.png
+      - install -Dm644 com.imageline.flstudio.32x32.png /app/share/icons/hicolor/32x32/apps/com.imageline.flstudio.png
+      - install -Dm644 com.imageline.flstudio.16x16.png /app/share/icons/hicolor/16x16/apps/com.imageline.flstudio.png
+    sources:
+      - type: file
+        path: com.imageline.flstudio.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html